python/cudf/cudf/core/column_accessor.py

```diff
@@ -341,6 +341,58 @@ def _pad_key(self, key: Any, pad_value="") -> Any:
             key = (key,)
         return key + (pad_value,) * (self.nlevels - len(key))
 
+    def replace_level_values(
+        self, mapping: dict or callable, level: int or None
```

Comment:
For the following to work, `Mapping`, `Any` and `Callable` must be imported from the `typing` module:

```suggestion
        self, mapping: [Mapping[Any, Any], Callable], level: int = None
```

Reply:
Should `level` be `Optional[int]` instead?